src/language-html/printer-html.js

```diff
@@ -98,6 +99,30 @@ function embed(path, print, textToDoc, options) {
             ? " "
             : line,
         ]);
+      } else if (isCustomBlock(node.parent, options)) {
+        const parser = inferScriptParser(node.parent, options);
+        let printed;
+        if (parser) {
+          try {
+            printed = textToDoc(node.value, { parser });
+          } catch (error) {
+            // Do nothing
+          }
+        } else {
+          const parsers = ["json", "yaml", "graphql"];
+          for (let i = 0; i < parsers.length && !printed; i++) {
+            const parser = parsers[i];
+            try {
+              printed = textToDoc(node.value, { parser });
```

Comment:
I don't like the idea guessing languages,  people even complaining about auto format with known language.

Reply:
I don't like this too, but in the real world custom blocks are often used without attributes.

Reply:
In that way, why don't we ignore it. Want format? Tell me what it is.

Reply:
I think formatting for custom blocks is needed. For example, [eslint-plugin-gridsome](https://github.com/gridsome/eslint-plugin-gridsome/blob/master/lib/rules/format-query-block.js) and [eslint-plugin-prettier-vue](https://github.com/meteorlxy/eslint-plugin-prettier-vue) use Prettier to format custom blocks.
Could you tell me what is the reason for ignoring it?

Reply:
I'm not saying disable formatting custom blocks.
I'm worried about wrong results.

We should ignore 
```html
<docs></docs>
```

But format
```html
<docs lang="markdown"></docs>
```

Maybe with some exception like `<page-query>` known as `graphql` block, like `<script>` we assume that's `javascript`

Reply:
I understood. What do you think about adding options for custom blocks like [eslint-plugin-prettier-vue](https://github.com/meteorlxy/eslint-plugin-prettier-vue)? (I'm reluctant to add options)

Reply:
I'll remove guessing languages for now. At least for this PR, it only supports custom blocks using the lang / type attribute.

Reply:
👍 , I guess we can use `--embedded-language-formatting` later?

Reply:
sounds good.